### PR TITLE
[MABL-6948] Address CVE-2022-29036 in Jenkins Integration (Jenkins Credentials)

### PR DESCRIPTION
**Pull request by simonychoy**
[MABL-6948](https://mabl.atlassian.net/browse/MABL-6948)
Bumps the `org.jenkins-ci.plugins:credentials` dependency to 2.6.1.1 to address CVE-2022-29036. 
Reported by Github - https://github.com/jenkinsci/mabl-integration-plugin/pull/59

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -121,6 +121,16 @@
...
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>structs</artifactId>
+                <version>1.23</version>
+            </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci</groupId>
+                <artifactId>symbol-annotation</artifactId>
+                <version>1.23</version>
+            </dependency>
```
Comment (simonychoy) [inline pom.xml:133]: I couldn't get the local build to run without adding these explicitly.

**Review by DonAtMabl — APPROVED** [review]
Looks good. Thanks!

File: pom.xml [hunk 2]
```diff
@@ -102,7 +102,7 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials</artifactId>
             <!-- see the Jenkins Enterprise compatibility note in the README before changing this version -->
-            <version>2.4</version>
+            <version>2.6.1.1</version>
```
Comment (twistedpair) [inline pom.xml:105]: Does this have any implications for Enterprise compatibility?

Comment (twistedpair) [reply pom.xml:105]: Looks like [this release](https://github.com/jenkinsci/mabl-integration-plugin#v0034-09042020) specifically addressed problems with the credentials plugin and Jenkins Enterprise. 

Were you able to follow the directions in the read me to verify that this new version works with Jenkins Enterprise?

Comment (simonychoy) [reply pom.xml:105]: I validated being able to generate deployments in mabl from the plugin on the Jenkins Enterprise image.

**Pull request merged**